apps/cms.c

```diff
@@ -897,8 +900,10 @@ int cms_main(int argc, char **argv)
                     goto end;
             }
 
-            res = EVP_PKEY_CTX_ctrl(pctx, -1, -1,
-                        EVP_PKEY_CTRL_CIPHER, EVP_CIPHER_nid(cipher), NULL);
+            ctx_params[0] = OSSL_PARAM_construct_utf8_string("cipher",
```

Comment:
This string would need to be defined properly as a #define and documented in a pod.

Reply:
Will do.

Reply:
Hopefully, it's enough or should it still be documented?